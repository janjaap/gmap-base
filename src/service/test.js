import Service from './';

describe('Service', () => {
    it('should match the snapshot', () => {
        const s = new Service();
        expect(s.events).toMatchSnapshot();
    });
});
