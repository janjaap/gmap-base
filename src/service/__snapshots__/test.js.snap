// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Service should match the snapshot 1`] = `
Array [
  "REQUEST_PENDING",
  "OK",
  "UNKNOWN_ERROR",
  "NOT_FOUND",
  "REQUEST_DENIED",
  "INVALID_REQUEST",
  "ZERO_RESULTS",
]
`;
